DOCSP-48592-point-users-to-local-atlas-experience (#109)
mongodb/docs-mongodb-vscode: source/connect.txt

   To create an |service| cluster using a Terraform template included
   with |vsce|, see :ref:`vsce-create-cluster-terraform`.

>   You can also use the Local Atlas Experience to build 

Reviewer: Could we also note that this feature is available for versions 1.13.0+ of MongoDB for VsCode. 

This version is in the process of releasing and corresponding version of AtlasCLI release will precede it. Are we able to hold on the publication of this update until then?

>    If you are a developer, consider using the Local Atlas Experience to build 
>    MongoDB Atlas deployments locally. Use the :atlascli:`Atlas CLI </>` 

Reviewer 2: [suggestion] for minimalism

```suggestion
   You can also use the Local Atlas Experience to build 
   MongoDB Atlas deployments locally. Use the :atlascli:`Atlas CLI </>` 
```

>    to set up and connect to local deployments with MongoDB for VS Code 

Reviewer 2: Just in the off chance that the product name changes, I would recommend using the constant here

```suggestion
   to set up and connect to local deployments with |vsce|
```

>    Starting in version 1.13.0 of |vsce|, you can also use the Local Atlas Experience 
>    to build MongoDB Atlas deployments locally. Use the :atlascli:`Atlas CLI </>` 
>    to set up and connect to local deployments with |vsce| directly from your 
>    command line. For more information, see 
>    :atlascli:`Manage Local and Cloud Deployments from the Atlas CLI </atlas-cli-local-cloud>`.
> 

Reviewer: Just a note on versions: users can create Local Atlas deployments and connect using the resulting connection string irregardless of vsce version.
For functionality to connect directly from command line, atlasCLI v1.42.0+ and vsce v1.13.0+ are required.

```suggestion
   You can also use the Local Atlas Experience 
   to build MongoDB Atlas deployments locally. Use the :atlascli:`Atlas CLI </>` 
   to set up local deployments. For more information, see 
   :atlascli:`Manage Local and Cloud Deployments from the Atlas CLI </atlas-cli-local-cloud>`.
   Starting in version 1.42.0 of the :atlascli:`Atlas CLI </>`  and version 1.13.0 of |vsce|, you can connect directly to your local deployments from your 
   command line.
```

>    directly from your command line. To create a Local Atlas deployment, 
>    see :atlascli:`Manage Local and Cloud Deployments from the Atlas CLI </atlas-cli-local-cloud>`.

Reviewer 2: [suggestion] for minimalism

```suggestion
   directly from your command line. For more information,, 
   see :atlascli:`Manage Local and Cloud Deployments from the Atlas CLI </atlas-cli-local-cloud>`.
```
